pymess/backend/dialer/daktela.py

```diff
@@ -125,3 +137,30 @@ def publish_message(self, message):
             # Do not re-raise caught exception. We do not know exact exception to catch so we catch them all
             # and log them into database. Re-raise exception causes transaction rollback (lost of information about
             # exception).
+
+    def create_message(self, recipient, content=None, related_objects=None, tag=None, template=None,
+                       is_predictive=False, **kwargs):
+        """
+        Create dialer message which will be logged in the database (content is not needed for this .
+        :param recipient: phone number of the recipient
+        :param related_objects: list of related objects that will be linked with the dialer message using generic
+        relation
+        :param tag: string mark that will be saved with the message
+        :param kwargs: extra attributes that will be saved with the message
+        """
+        if not (is_predictive or content):
+            raise PersistenceException(_('Autodialer message must contain content.'))
```

Comment:
tohle má být na modelu. Raisovat tady PersistenceException je nesmysl

Reply:
tohle stále platí

Reply:
moved to model